xt-bundle-aos.cfg

```diff
@@ -0,0 +1,53 @@
+[path]
+# this shall be fast drive for doing most of the build on
+workspace_base_dir = /data/storage/work/xen_prod/build
+
+[bundle_config]
+top_template = bundle_templates/top_template.json
+boardModel = hw1
+formatVersion = 1
+components = dom0 domd domf
+ostree_path = /data/.aos_ostree
+bundle_deploy_dir = /data/bundles/
+
+[dom0]
+enabled = 1
```

Comment:
Why we need this flag? It duplicates `components` field, Doesn't it?

Reply:
Added this because, from my standpoint, using flipswitch is more useful then add/remove component name. 
Also in that case it's more protection from spelling mistakes, such as dmof instead of domf

Reply:
Remove components field then.